### DISCUSSION TITLE
POLL: What scripting language do you use the most?

### DISCUSSION THREAD

**Discussion by drkameleon** [arturo-lang/arturo#1006] [Polls]
Let's say you need to code some script fast, for some thing trivial: you want to mass rename a bunch of files, download/process/parse a webpage, write some fun little game, you name it...

**The question is:** if you have some language of preference that you would normally resort to, on a daily basis, which one would it be?

**Comment by retsyo** [comment]
aha, what is the "other" language which occupies 50% of the answer

**Reply by RickBarretto** [reply]
That is a good question, lol!
